Ryujinx.HLE/HOS/Services/Time/StaticService/ITimeZoneServiceForGlue.cs

```diff
@@ -0,0 +1,142 @@
+using ARMeilleure.Memory;
+using Ryujinx.Common;
+using Ryujinx.Common.Logging;
+using Ryujinx.HLE.HOS.Services.Time.TimeZone;
+using System;
+using System.Text;
+
+namespace Ryujinx.HLE.HOS.Services.Time.StaticService
+{
+    class ITimeZoneServiceForGlue : IpcService
+    {
+        private TimeZoneContentManager  _timeZoneContentManager;
```

Comment:
```suggestion
        private TimeZoneContentManager _timeZoneContentManager;
```
and same for next vars.